tasks/browser/agent/agent.go

```diff
@@ -9,6 +9,6 @@ import (
 func RegisterWith(registrationFunc func(tasks.Task, bool)) {
 	log.Debug("Registering Browser/Agent/*")
 
-	registrationFunc(BrowserAgentDetect{}, false)
-	registrationFunc(BrowserAgentGetSource{}, false)
+	registrationFunc(BrowserAgentDetect{}, true)
```

Comment:
I can't believe this is the first time we enable these task by default but it was about time